scanner/cmd/scannerctl/main.go

```diff
@@ -12,82 +12,140 @@ import (
 	"strings"
 
 	"github.com/google/go-containerregistry/pkg/authn"
+	"github.com/spf13/cobra"
 	"github.com/stackrox/rox/pkg/mtls"
 	"github.com/stackrox/rox/pkg/utils"
 	"github.com/stackrox/rox/scanner/indexer"
+	"github.com/stackrox/rox/scanner/internal/version"
 	"github.com/stackrox/rox/scanner/pkg/client"
 	"golang.org/x/sys/unix"
 )
 
-var authEnvName = "ROX_SCANNERCTL_BASIC_AUTH"
+type rootCommand struct {
+	cobra.Command
+	ScannerClient client.Scanner
+}
 
-func main() {
-	address := flag.String("address", ":8443", "Address of the scanner service.")
-	serverName := flag.String("server-name", "scanner-v4.stackrox",
+func rootCmd(ctx context.Context) *cobra.Command {
+	cmd := rootCommand{
+		Command: cobra.Command{
+			Use:          "scannerctl",
+			Version:      version.Version,
```

Comment:
I expected this to add a `--version` flag, but I don't see it. Do you?

Reply:
`version.Version` is empty?

Reply:
Can you confirm if you're getting empty version strings and resolve this? Corba won't add the version flag if the version is empty.